cmd/main.go

```diff
@@ -47,25 +49,26 @@ func main() {
 	// TODO: Remove this once the config-map based mount approach lands after rebase
 	oldServiceCASrc := fmt.Sprintf("%s/service-ca.crt", builder.SecretCertsMountPath)
 	oldServiceCADst := fmt.Sprintf("%s/service.crt", tlsCertRoot)
-	err = CopyIfExists(oldServiceCASrc, oldServiceCADst)
+	err = CopyFileIfExists(oldServiceCASrc, oldServiceCADst)
 	if err != nil {
 		fmt.Printf("Error setting up service CA cert: %v", err)
 		os.Exit(1)
 	}
 
+	// TODO: Remove this once the build controller mounts the internal registry's CA
+	// in certs.d/<internal-registry-hostname>/
 	newServiceCASrc := fmt.Sprintf("%s/service-ca.crt", builder.ConfigMapCertsMountPath)
 	newServiceCADst := fmt.Sprintf("%s/openshift-service.crt", tlsCertRoot)
-	err = CopyIfExists(newServiceCASrc, newServiceCADst)
+	err = CopyFileIfExists(newServiceCASrc, newServiceCADst)
```

Comment:
and this, presumably, "might" exist depending on the raciness of the creation/population of the configmap that was created by the build controller.

Reply:
and this is only working for us because the configmap is getting populated fast enough